model/clusters_mgmt/v1/upgrade_policy_resource.model

```diff
@@ -29,4 +29,9 @@ resource UpgradePolicy {
 	// Deletes the upgrade policy.
 	method Delete {
 	}
+
+	// Reference to the state of the upgrade policy.
+	locator State {
```

Comment:
Don't you want to add it to `update_policy_type.model` as well?

Reply:
No.

There's a design discussion around it.